### ISSUE TITLE
real-time updates seem to stop functioning after some time

### ISSUE THREAD

**Issue by kabaka** [taigaio/taiga-front#655]
This takes place on tree.taiga.io. I'm not sure about self-hosted.

The kanban view was correctly updating in real-time. Later, I noticed it was no longer updating. Looking at the console, I found:

```
angular.js:12221 Unsubscribe from: changes.project.70547.userstories
```

and:

```
libs.js:14 WebSocket closed.
```

To see if maybe I was just on a page which does not support real-time updates, I navigated back to the kanban page. This was logged:

```
libs.js:14 Subscribe to: changes.project.70547.userstories
```

But the websocket is never re-opened, and never seems to re-open until refreshing.

This is consistent, but the exact steps to reproduce it aren't immediately apparent. I will update this issue if I learn anything else.

**Comment by superalex** [comment]
Thank you very much for reporting @kabaka it sounds like a bug related to taiga-events (https://github.com/taigaio/taiga-events), we will try to reproduce it too.

Regards!,

**Comment by nonameolsson** [comment]
What's the status on this issue?

**Comment by juanfran** [comment]
It should have been fixed months ago. Do you have this issue? in tree.taiga.io or in your own instance?

**Comment by nonameolsson** [comment]
Maybe it's another issue.. If i create a card in Kanban it shows up in other browser in real-time. But if I start to move it around it doesn't keep up. The red warning message shows up saying:
```
Oops, something happened...
Someone inside Taiga has changed this before and our Oompa Loompas cannot apply your changes. Please reload and apply your changes again (they will be lost).
```
In the console the following message shows up:
`PATCH https://api.taiga.io/api/v1/userstories/1063798?include_attachments=true&include_tasks=true 400 (Bad Request)``
Maybe this is another issue? I searched among the issues and found this one and thought it was the same problem that I experienced.
